bson/binary.py

```diff
@@ -364,4 +364,7 @@ def __ne__(self, other: Any) -> bool:
         return not self == other
 
     def __repr__(self) -> str:
-        return f"Binary({bytes.__repr__(self)}, {self.__subtype})"
+        if self.__subtype == SENSITIVE_SUBTYPE:
+            return f"Binary('<redacted>', {self.__subtype})"
```

Comment:
Since this is no longer an accurate representation of the object we need to surround it with <> so that repr+eval() will raise an error instead of silently changing the data, like this:
```
return f"<Binary(REDACTED, {self.__subtype})>"
```

Reference: https://docs.python.org/3/library/functions.html#repr

Reply:
I added an entry to the changelog. Since the the jira is tagged `Fixes 4.7.3` I assumed that is the release. I don't know how to get a link to the release notes from jira so I didn't include the link.

Reply:
Thanks!  I added the link.